3 way heaven

intrvalio opened the pull request.

Linglingpo committed:
Digital Interrupt enable/disable

Linglingpo committed:
Trying to implement  an array for Config input and output (not working)

problem on setting the default array into parameter

Commit:
Cleaning Up - Will make a few more changes tomorrow morning...

Commit:
shall return later...

Commit:
hmmm. talk later...

Linglingpo committed:
Basic state for all pins (all input) works

Linglingpo committed:
Set and Get state from the library

Commit:
Get Library State access complete.

Commit:
Small thing - just pushed...

Linglingpo committed:
Write comment and testing

Linglingpo committed:
Reading state in Arduino and send msg when press bt

Commit:
Start working on Communicate class

Before Milin Breaking the code

Commit:
For Tuesday

Commit:
Working with Processing is like bathing a cat!

Commit:
Still fighting with the wet cat that Processing is. Discovery working!

Commit:
Corrected Spelling.

Commit:
Shuffled some items around - hopefully for the better ;)

Commit:
Another shuffle.

Commit:
Watch out for the Unidentified Flying Objects

Commit:
More undisclosed unidentified flying objects!

Commit:
BROKEN!!!!!

Commit:
GREMILNS...

Commit:
changing...

Commit:
MORE LIKE 3-WAY-HELL

Linglingpo committed:
Don't know where to put the reConstructPreamble

Problem with SerialEvent

Commit:
oops

Commit:
Meow!

Commit:
Making Progress - Discovery is 90% Done...

Commit:
DICSOVERY !

ghost commented:
DISCOVERY CODE - Working with Processing. Needs some optimisation (dryness) and further modularization and further completion of the 3-way handshake for HARDSERIAL

Linglingpo committed:
Bluetooth sucks !

Working on transmit / transmission function in Communicate.cpp, Not
working yet.

Linglingpo committed:
Not working ... :(

/Users/milin/Documents/Arduino/libraries/LetsCommunicate/Communicate.h:9
5:3: error: 'transmission' does not name a type
   transmission * transmitState;

??????? WHY !!

Linglingpo committed:
fix little thing...

Linglingpo committed:
Data constructer ok

Linglingpo committed:
Clean up & change "action" variable name to "payloadType"

Linglingpo committed:
constructMaster (Preamble + Data) ok

Linglingpo committed:
Fix digiSize

14 = 0x0E //14 Digital Pins

Linglingpo committed:
Question mark hell...

DIG - ok , DXT - Not ok, ANA - Not ok
SYN - ok, FIN - ok, SNY - Not ok

Linglingpo committed:
small changes

Linglingpo committed:
Rename variables and clean up

Arduino compiled (Haven’t test it yet because I forgot to bring the
Arduino home)
